.github/workflows/cleanup.yaml

```diff
@@ -34,3 +34,24 @@ jobs:
       continue-on-error: true
       run: |
         kubectl delete ns ${PREVIEW_NAMESPACE}
+
+  webui-cleanup:
+    runs-on: ubuntu-latest
+    if: (github.event.action == 'unlabeled' && github.event.label.name == 'webui-preview') || contains(github.event.pull_request.labels.*.name, 'webui-preview')
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Prepare environment
+        run: |
+          echo "BRANCH_NAME=${GITHUB_HEAD_REF##*/}" >> $GITHUB_ENV
+
+      - name: Web UI preview
+        uses: nuxeo/ui-team-gh-actions/trigger-workflow@bcc698511e6d9ce90ea120a6ee2cdde82079cde4
+        with:
+          owner: nuxeo
+          repo: nuxeo-web-ui
+          workflow-id: remote-clean.yaml
```

Comment:
Once https://github.com/nuxeo/nuxeo-web-ui/pull/1436 is merged, we need to update any reference to the workflows `remote-{clean,preview}` to just `{clean,preview}`.